src/consumerIncentives/slice.ts

```diff
@@ -76,15 +78,25 @@ const slice = createSlice({
     }),
   },
   extraReducers: (builder) => {
-    builder.addCase(
-      AppActions.UPDATE_REMOTE_CONFIG_VALUES,
-      (state, action: UpdateConfigValuesAction) => {
-        state.superchargeV2Enabled = action.configValues.superchargeV2Enabled
-        state.superchargeRewardContractAddress =
-          action.configValues.superchargeRewardContractAddress
-        state.superchargeV1Addresses = action.configValues.superchargeV1Addresses
-      }
-    )
+    builder
+      .addCase(REHYDRATE, (state, action: RehydrateAction) => ({
+        ...state,
+        ...getRehydratePayload(action, 'supercharge'),
+        loading: false,
+        error: false,
+        fetchAvailableRewardsLoading: false,
+        fetchAvailableRewardsError: false,
+        availableRewards: [],
```

Comment:
Site note: why don't we keep the fetched available rewards? At least for **some** time? Seems wasteful to query on every app start, right?